src/controllers/vars_desktop.scss

```diff
@@ -59,4 +59,9 @@ $notification-width: 12.5rem !default;
   --footer-panel-height: 12.5rem;
   --footer-panel-height-lidarresult: 21.88rem;
   --current-footer-panel-height: 0;
+
+  // LIDAR profile
+  --app-margin: 0.62rem;
+  --lidarprofile-height: 21.88rem;
+  --profile-legend-width: 9.38rem;
```

Comment:
--footer-panel-height-lidarresult  should be in this section why --app-margin is in this section?

Reply:
the CSS I added used this variable, I wanted to keep the code as similar as the initial one for the beginning.

Reply:
Yes, but it's a common variable already defined as SCSS variable then it should be 
```
--app-margin: #{$app-margin};
```
some lines above :-)

Reply:
Why don't you want to put `--footer-panel-height-lidar` after the LIDAR comment?

Reply:
Thanks :-)